### PR TITLE
LLM: GPU Example Updates for Windows

### PR DESCRIPTION
**Pull request by JinBridger**
Updates all examples under HF-Transformers-AutoModels and PyTorch Model.

### CODE REVIEW

**Comment by Oscilloscope98** [comment]
We may have some verification on those models that are somehow different, e.g. whisper, distil-whisper; as well as voice assistant examples, maybe also Qwen-VL and llava with image input before merged.

**Comment by JinBridger** [comment]
Looks like voice assistant can run on Windows after remove `import soundevice` in script.

Also, it looks like `soundevice` is not needed on Windows. I config the environment without install it but the script still can run.

I've added the modification for voice assistant again.

**Comment by Oscilloscope98** [comment]
> Looks like voice assistant can run on Windows after remove `import soundevice` in script.
> 
> Also, it looks like `soundevice` is not needed on Windows. I config the environment without install it but the script still can run.
> 
> I've added the modification for voice assistant again.

@plusbang Do you have an idea why we need this `sounddevice` before? :)

**Comment by plusbang** [comment]
> > Looks like voice assistant can run on Windows after remove `import soundevice` in script.
> > Also, it looks like `soundevice` is not needed on Windows. I config the environment without install it but the script still can run.
> > I've added the modification for voice assistant again.
> 
> @plusbang Do you have an idea why we need this `sounddevice` before? :)

I‘m not quite sure :( .....But if we could run on GPU (linux and windows OS) without `sounddevice`, I think we could remove it.

**Review by plusbang — APPROVED** [review]
LGTM

File: python/llm/example/GPU/HF-Transformers-AutoModels/Model/voiceassistant/generate.py [hunk 1]
```diff
@@ -92,6 +92,8 @@ def get_prompt(message: str, chat_history: list[tuple[str, str]],
     whisper.config.forced_decoder_ids = None
     whisper = whisper.to('xpu')
     
+    # When running LLMs on Intel iGPUs for Windows users, we recommend setting `cpu_embedding=True` in the from_pretrained function.
```
Comment (Oscilloscope98) [inline python/llm/example/GPU/HF-Transformers-AutoModels/Model/voiceassistant/generate.py:97]: lets change LLM -> Llama models here, to differentiate from whisper models

File: python/llm/example/GPU/PyTorch-Models/Model/distil-whisper/recognize.py [hunk 2]
```diff
@@ -51,6 +51,7 @@
     audio = dataset[0]["audio"]
 
     model = AutoModelForSpeechSeq2Seq.from_pretrained(model_path)
+
```
Comment (Oscilloscope98) [inline python/llm/example/GPU/PyTorch-Models/Model/distil-whisper/recognize.py:54]: Remove  the line here :)

**Comment by Oscilloscope98** [comment]
Please rebase this PR to see whether there are still some models we have not add windows related info yet :)

File: python/llm/example/GPU/HF-Transformers-AutoModels/Model/voiceassistant/generate.py [hunk 3]
```diff
@@ -20,6 +20,8 @@
 import argparse
 import numpy as np
 import inquirer
+
+# for Windows device, please remove `import sounddevice`
```
Comment (Oscilloscope98) [inline python/llm/example/GPU/HF-Transformers-AutoModels/Model/voiceassistant/generate.py:24]: For Windows users...

**Comment by Oscilloscope98** [comment]
Hi @jason-dai,

This is the PR to add windows supports to all models in HF-Transformers-AutoModels and PyTorch-Models examles.

We have chosen some special models to do some verification: whisper, distil-whisper, voice assistant, llava and qwen-vl for running on Windows, in which only qwen-vl has some problem (https://github.com/analytics-zoo/nano/issues/1009) but it does not related to Windows, so we still added the Windows support info for qwen-vl models.

We are merging this PR for now, and we may have more verification to other models later. Please let us know for any comments. Thank you! :)

**Pull request merged**